src/V3Ast.h

```diff
@@ -921,17 +925,28 @@ class VVarType final {
         static const char* const names[]
             = {"?",          "GPARAM",   "LPARAM",  "GENVAR",   "VAR",   "SUPPLY0", "SUPPLY1",
                "WIRE",       "WREAL",    "TRIWIRE", "TRI0",     "TRI1",  "PORT",    "BLOCKTEMP",
-               "MODULETEMP", "STMTTEMP", "XTEMP",   "IFACEREF", "MEMBER"};
+               "MODULETEMP", "STMTTEMP", "XTEMP", "IFACEREF", "MEMBER", "WOR", "TRIOR",
+               "WAND", "TRIAND"};
         return names[m_e];
     }
     bool isParam() const { return m_e == GPARAM || m_e == LPARAM; }
     bool isSignal() const {
         return (m_e == WIRE || m_e == WREAL || m_e == TRIWIRE || m_e == TRI0 || m_e == TRI1
-                || m_e == PORT || m_e == SUPPLY0 || m_e == SUPPLY1 || m_e == VAR);
+                || m_e == PORT || m_e == SUPPLY0 || m_e == SUPPLY1 || m_e == VAR || m_e == WOR
+                || m_e == TRIOR || m_e == WAND || m_e == TRIAND);
     }
     bool isNet() const {
         return (m_e == WIRE || m_e == TRIWIRE || m_e == TRI0 || m_e == TRI1 || m_e == SUPPLY0
-                || m_e == SUPPLY1);
+                || m_e == SUPPLY1 || m_e == WOR || m_e == TRIOR || m_e == WAND || m_e == TRIAND);
+    }
+    bool isWor() const {
+        return (m_e == WOR || m_e == TRIOR);
+    }
+    bool isWand() const {
+        return (m_e == WAND || m_e == TRIAND);
```

Comment:
IEEE says WOR is exactly same as TRIOR, so just remove WOR and create it as TRIOR.  Likewise WAND.